datafusion/core/tests/sql/path_partition.rs

```diff
@@ -57,55 +53,6 @@ use object_store::{
 use object_store::{Attributes, MultipartUpload, PutMultipartOpts, PutPayload};
 use url::Url;
 
-#[tokio::test]
-async fn parquet_partition_pruning_filter() -> Result<()> {
```

Comment:
can you remind me again why this test was removed ?

Reply:
It was relying on `ListingTable` to do the pushdown without the optimizer pass.
I felt that it was better suited for an `.slt` test that is fully wired up, so I essentially moved it: https://github.com/apache/datafusion/pull/15769/files#r2074193501